docs/source/components/hooks/use-organizations.md

```diff
@@ -7,16 +7,33 @@ summary: Hook that returns organizations.
 
 ```jsx
 import React from 'react';
-import { useOrganizations } from '@availity/hooks';
-// ...
+import { useCurrentUser, useOrganizations } from '@availity/hooks';
+
 const Component = () => {
-  const { data, status, error } = useOrganizations();
+  const { data: user } = useCurrentUser();
+  const { data, isFetching } = useOrganizations(
+    { params: { permissionId: ['5'], userId: user?.id } },
+    { enabled: !!user?.id }
+  );
 
   return (
     <div>
-      {status === 'loading' ? 'Loading...' : data.data.organizations[0].name}
+      {isFetching
+        ? 'Loading...'
+        : data?.data?.organizations?.[0]?.name || 'No organizations found'}
     </div>
   );
 };
-// ...
 ```
+
+## Props
+
+## `config: AxiosRequestConfig`
+
+Config passed to the `getOrganizations` call from `@availity/api-axios`.
```

Comment:
it would be useful to link to this: https://availity.github.io/sdk-js/api/getting-started/#config-1

Reply:
Added